.github/test/cd-agent/projects/example.yaml

```diff
@@ -17,11 +17,15 @@ cohortSelector:
     # Identifier system filter. Only patients with identifiers from this system will be included in the cohort.
     patientIdentifierSystem: "https://ths-greifswald.de/fhir/gics/identifiers/Pseudonym"
     # Policy system filter. Only policies with identifiers from this system will be considered valid.
+    # see e.g.: https://simplifier.net/medizininformatikinitiative-modulconsent/2.16.840.1.113883.3.1937.777.24.5.3--20210423105554
     # policySystem: "MII CS Consent Policy"
     policySystem: "urn:oid:2.16.840.1.113883.3.1937.777.24.5.3"
```

Comment:
We could write `policySystem: "urn:oid:2.16.840.1.113883.3.1937.777.24.5.3" # MII CS Consent Policy`

But it's also ok as it is